### PR TITLE
FEATURE: Add `solved_topics_auto_close_hours` site setting.

### PR DESCRIPTION
**Pull request by tgxworld**
https://meta.discourse.org/t/automatically-close-solved-topics-after-n-hours/58383

### CODE REVIEW

**Comment by tgxworld** [comment]
@SamSaffron  Can you review this? Thank you

File: config/locales/server.en.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ en:
     accept_all_solutions_trust_level: "Minimum trust level required to accept solutions on any topic (even when not OP)"
     empty_box_on_unsolved: "Display an empty box next to unsolved topics"
     solved_quote_length: "Number of characters to quote when displaying the solution under the first post"
+    solved_topics_auto_close_hours: "Auto close topic (n) hours after topic has been marked as solved"
```
Comment (tgxworld) [inline config/locales/server.en.yml]: Let me know if this name for the site setting is :ok_hand:

**Comment by davidtaylorhq** [comment]
If a post is "un-selected" as the solution, will the timer be cancelled?

**Comment by SamSaffron** [comment]
did we address all the comments from the meta topic?

**Comment by tgxworld** [comment]
Yup I did in https://meta.discourse.org/t/automatically-close-solved-topics-after-n-hours/58383/8?u=tgxworld. I didn't agree with some of the comments.

**Comment by SamSaffron** [comment]
Go ahead and merge :)

**Pull request merged**